wingtips-core/src/test/java/com/nike/wingtips/http/HttpRequestTracingUtilsTest.java

```diff
@@ -260,6 +260,51 @@ public void fromRequestWithHeaders_sets_sampleable_to_attribute_value_if_sampled
         assertThat(newSpan.isSampleable()).isFalse();
     }
 
+    @DataProvider(value = {
```

Comment:
There's an already-existing test that verifies missing/empty sampled header coerces to true: `fromRequestWithHeaders_sets_sampleable_to_true_if_sampled_header_is_null_or_missing_from_both_headers_and_attributes`.

There's a separate mechanism for determining whether new traces are sampled: `RootSpanSamplingStrategy`.